src/fmu/dataio/dataio.py

```diff
@@ -378,7 +378,7 @@ class ExportData:
     classification: Optional[str] = None
     config: dict = field(default_factory=dict)
     content: Optional[Union[dict, str]] = None
-    depth_reference: str = "msl"  # deprecated
+    depth_reference: Optional[str] = None  # deprecated
```

Comment:
This should have been done in #624 . Now it emits a deprecation warning every time ExportData is initialized 😱